conf/local.conf.sample

```diff
@@ -91,10 +91,10 @@ USER_FEATURES += "~3g"
 USER_FEATURES += "~nfc"
 
 # How many tasks bitbake should run in parallel
-BB_NUMBER_THREADS ?= "${@int(oe.utils.cpu_count() * 1.5)}"
+# BB_NUMBER_THREADS ?= "${@int(oe.utils.cpu_count())}"
 
 # How many jobs 'make' should run in parallel when compiling
-PARALLEL_MAKE ?= "-j ${@oe.utils.cpu_count() * 2}"
+# PARALLEL_MAKE ?= "-j ${@oe.utils.cpu_count()}"
```

Comment:
Drop the space after the # in these variable commenting cases? I guess that's how we do it for such stuff.

Reply:
Fixed, thanks